Resources/Prototypes/_LostParadise/Maps/kerberos.yml

```
@@ -0,0 +1,77 @@
+- type: gameMap
```

yamllint

[error] wrong new line character: expected \n

(new-lines)